ap/settings/common.py

```diff
@@ -81,6 +82,27 @@
     "django_structlog.middlewares.RequestMiddleware",
 ]
 
+# -- Sentry error tracking
+
+if os.environ.get("SENTRY_DSN"):
+    # Third-party
+    import sentry_sdk
+    from sentry_sdk import set_tag
+    from sentry_sdk.integrations.django import DjangoIntegration
+
+    sentry_sdk.init(
+        dsn=os.environ["SENTRY_DSN"],
+        environment=ENV,
+        integrations=[DjangoIntegration()],
+        traces_sample_rate=0.0,
+        send_default_pii=True,
+    )
+
+    if "runworker" in sys.argv:
```

Comment:
Do we need this block? Unsure how useful this is

Reply:
From line 101 down

Reply:
Agreed, lets remove it